src/app/util/binding-table.h

```diff
@@ -75,19 +76,44 @@ class BindingTable
 
     Iterator end();
 
+    void SetPersistentStorage(PersistentStorageDelegate * storage) { mStorage = storage; }
+
+    CHIP_ERROR LoadFromStorage();
+
     static BindingTable & GetInstance() { return sInstance; }
 
 private:
     static BindingTable sInstance;
+    static constexpr uint8_t kEntryStorageSize    = 64;
+    static constexpr uint8_t kListInfoStorageSize = 16;
```

Comment:
Where are these constants coming from?

Should these be using `TLV::EstimateStructOverhead`?